### PR TITLE
Sphinx calls (chat rows and Jitsi meet integration)

### PR DESCRIPTION
**Pull request by tomastiminskas**

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -117,6 +117,9 @@ ext.deps = [
     viewBindingDelegateNoReflect: "com.github.kirich1409:viewbindingpropertydelegate-noreflection:1.4.7",
     giphy: [
             sdk: "com.giphy.sdk:ui:2.1.2"
+    ],
+    jitsi: [
+            sdk: "org.jitsi.react:jitsi-meet-sdk:2.+"
```
Comment (05nelsonm) [inline gradle/dependencies.gradle:122]: Peg to a version number instead of using 2.+

**Review by 05nelsonm — APPROVED** [review]
Looks great!

I do, however, wonder if the separate `bottom-menu-call` module is even necessary as it's only accessible from a chat, thus being better served in the `chat-common` module directly utilizing the `bottom-menu` module's builder there instead of creating unnecessary interfaces/handler classes (that hold only the viewstate container and do nothing else).

**Comment by tomastiminskas** [comment]
@05nelsonm don't merge it and I will do the change you suggested for the menu-bottom-call. Will remove the module and use menu-bottom.

**Comment by tomastiminskas** [comment]
@05nelsonm changes pushed:
- Jitsi Meet sdk pegged to version number
- `bottom-menu-call` module removed

**Pull request merged**